gemrb/GUIScripts/PortraitWindow.py

```diff
@@ -16,7 +16,224 @@
 # Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
 #
 
-import GUICommonWindows
+import GemRB
+import GUICommon
 
-def UpdatePortraitWindow():
-	GUICommonWindows.UpdatePortraitWindow()
+from GUIDefines import *
+from GUICommonWindows import *
```

Comment:
Let's avoid imports like this, since they pollute the namespace. It took quite some effort to remedy this in the past.
Of course it's fine for constant-only files like GUIDefines.py.

Reply:
yeah, its not intended to be permanent, and I just kinda forgot about it. I just wanted to be able to copy and paste the original code without changing it at the same time.